pkg/updater/eval.go

```diff
@@ -231,3 +253,37 @@ func evalProperties(rule *evalpb.Rule, testResult TestResult) *tspb.TestStatus {
 	want := rule.GetComputedStatus()
 	return &want
 }
+
+func evalTargetProperties(rule *evalpb.Rule, targetResult TargetResult) *tspb.TestStatus {
+	for _, cmp := range rule.TestResultComparisons {
+		if cmp.Comparison == nil {
+			return nil
+		}
+		var stscmp func(tspb.TestStatus, tspb.TestStatus) bool
+		stsval := cmp.Comparison.GetTargetStatusValue()
+		switch cmp.Comparison.Op {
```

Comment:
If you don't plan immediately to implement other comparison operations (and I think equals alone is fine), make this a simple if statement that returns early.

```
// Only EQ is supported
if cmp.Comparision.Op != evalpb.Comparision_OP_EQ {
    return nil
}
```